tests/backpressure_test/Makefile

```diff
@@ -33,8 +36,6 @@ CHANS ?= 1 4
 RX_TX_INCS ?= 5;5 10;0 0;10
 BITDEPTHS ?= 8 16 32
 
-XCC_FLAGS_XCORE_AI = $(SHARED_FLAGS) -DSAMPLE_FREQUENCY=768000 -DNUM_I2S_LINES=4 -DRECEIVE_DELAY_INCREMENT=5 -DSEND_DELAY_INCREMENT=5
```

Comment:
What test coverage, if any, has been removed by deleting these lines?  What do the `RECEIVE_DELAY_INCREMENT` and `SEND_DELAY_INCREMENT` constants affect?

Reply:
No coverage - this line in the Makefile was for a now-redundant test where this test and this test alone was compiled for XS3 and run on xsim. Given all tests are now run for XS3 on xsim, this is a redundant config.

(those constants are compile-time constants for the test XE - the XE runs multiple times incrementing the amount of delay in the send/recieve callbacks until something breaks, simulating longer and longer time spent in these callbacks. The test aim is to make sure that our implementation isn't taking so long to do its work to meet timing that the customer can't do anything useful in the remaining time.)